src/swarm/client/RequestSetup.d

```diff
@@ -718,6 +718,24 @@ public template Suspendable ( )
 
         params_.suspend_register = this.suspend_register;
     }
+
+    /***************************************************************************
+
+        Copies the value of the suspend unregister member into the provided
+        request params class instance.
+
+        Params:
+            params = IRequestParams instance to write into
+
+    ***************************************************************************/
+
+    private void setup_suspend_unregister ( IRequestParams params )
```

Comment:
Was this removed by accident or missing?

Reply:
Turns out this was accidentally removed by me in https://github.com/sociomantic-tsunami/swarm/commit/6e26ffa.

Reply:
This method had a comment `// deprecated: remove in swarm v6`.
This means that the commit adding this in this PR should be removed, and dhtproto should be fixed instead?

Reply:
Looks to me like it meant "remove the static if". (That's without looking into things more deeply.)

Reply:
If it'd been me writing that deprecation comment (which it most likely was), I would have written it in the method doc if I'd meant to remove the whole method ;)

Reply:
The tests in dhtproto pass now, with the static if removed.

Reply:
For now, I have updated the commit description to better indicate the change.